editor.planx.uk/src/@planx/components/TextInput/Public.tsx

```diff
@@ -54,9 +53,13 @@ const TextInputComponent: React.FC<Props> = (props) => {
               else if (type === "phone") return "tel";
               return "text";
             })(props.type)}
-            multiline={props.type && ["long", "extraLong"].includes(props.type)}
+            multiline={
+              getTextLimit(props.type, props.customLength) > 120 &&
+              props.type !== "email"
+            }
             rows={
-              props.type && ["long", "extraLong"].includes(props.type)
+              getTextLimit(props.type, props.customLength) > 120 &&
```

Comment:
nit: can you reuse `characterCountLimit` definition above rather than repeating `getTextLimit(props.type, props.customLength)` here?